dkg/dkg.go

```diff
@@ -25,16 +24,32 @@ type Actor interface {
 	// setup has not been done.
 	GetPublicKey() (kyber.Point, error)
 
+	// Encrypt encrypts the given message into kyber points
```

Comment:
I can see (here and below) that we're returning kyber points.
What do they contain ? What's K and C? Which one's the encrypted message and which one is the other stuff (randomness, etc.) ? 😅

Reply:
added description of K and C